deployment-configuration/compose/templates/auto-compose.yaml

```diff
@@ -0,0 +1,241 @@
+version: '3.7'
+
+services:
+  traefik:
+    image: "traefik:v2.10"
+    container_name: "traefik"
+    networks:
+    - ch
+    command:
+      - "--log.level=INFO"
+      - "--api.insecure=true"
+      - "--providers.docker=true"
+      - "--providers.docker.exposedbydefault=false"
+      - "--entrypoints.web.address=:80"
+      - "--entrypoints.websecure.address=:443"
+      - "--providers.file.directory=/etc/traefik/dynamic_conf"
+    ports:
+      - "80:80"
+      - "443:443"
+    volumes:
+      - "/var/run/docker.sock:/var/run/docker.sock:ro"
+      - "./certs/:/certs/:ro"
+      - "./traefik.yaml:/etc/traefik/dynamic_conf/conf.yml:ro"
+
+{{- range $app_name, $app_config := .Values.apps }}
+  {{- if has $app_name (list "argo" "events" "nfsserver") -}}
+  {{- continue -}}
+  {{- end -}}
+  {{ $deployment :=  $app_config.harness.deployment }}
+  {{- if eq $app_name "nfsserver" }}
```

Comment:
better remove, there's no use case of the nfs server for docker-compose (its purpose is to provide a ReadWriteMany volume on K8s, but all volumes are equivalent of ReadWriteMany on docker as we are on a single node)